### DISCUSSION TITLE
Custom URL for update

### DISCUSSION THREAD

**Discussion by Romain-GABET** [arkypita/LaserGRBL#2309] [Q&A] [answered]
Hi everyone,

I've forked this project and I want the software to be able to fetch updates from my forked repository. How can I do that? How can I set a specific URL to get new update?

**Comment by arkypita** [comment, marked as answer]
just set `public static string UpdateMain = "https://api.github.com/repos/<yourname>/<yourrepo>/releases";`

**Comment by Romain-GABET** [comment]
Oh okay rather simple. Thanks !
